commit 959773138ac9e9c7f2ebc79f8abcec661b76c9bb
Author: Geert Uytterhoeven <geert+renesas@glider.be>
Date:   Thu Jan 10 14:39:16 2019 +0100

    arm64: dts: renesas: r8a77965: Enable DMA for SCIF2
    
    [ Upstream commit 05c8478abd485507c25aa565afab604af8d8fe46 ]
    
    SCIF2 on R-Car M3-N can be used with both DMAC1 and DMAC2.
    
    Fixes: 0ea5b2fd38db56aa ("arm64: dts: renesas: r8a77965: Add SCIF device nodes")
    Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index 0cd44461a0bd..f60f08ba1a6f 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -951,6 +951,9 @@
 				 <&cpg CPG_CORE R8A77965_CLK_S3D1>,
 				 <&scif_clk>;
 			clock-names = "fck", "brg_int", "scif_clk";
+			dmas = <&dmac1 0x13>, <&dmac1 0x12>,
+			       <&dmac2 0x13>, <&dmac2 0x12>;
+			dma-names = "tx", "rx", "tx", "rx";
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 310>;
 			status = "disabled";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1069_959773138ac9e9c7f2ebc79f8abcec661b76c9bb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1069_959773138ac9e9c7f2ebc79f8abcec661b76c9bb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
